src/components/components/Mixins.js

```diff
@@ -42,7 +42,7 @@ export default class Mixin extends React.Component {
 
   updateMixins = (value) => {
     const entity = this.props.entity;
-
+    value = Array.isArray(value) ? value : [value];
```

Comment:
what is this for? @ihordubas1

Reply:
reverted

Reply:
this looks like a way to handle a case where a non-array is passed; that shouldn't happen with the new version, reverting

Reply:
@kfarr 
can you check the new PR (https://github.com/3DStreet/3dstreet-editor/pull/183) 
can we close the current PR and use #183?